core/src/test/java/org/infinispan/functional/decorators/FunctionalAdvancedCache.java

```diff
@@ -274,32 +274,32 @@ public void start() {
 
    @Override
    public void addInterceptor(CommandInterceptor i, int position) {
-      // TODO: Customise this generated block
+      cache.addInterceptor(i, position);
```

Comment:
I figured I shouldn't implement the interceptor-related part of the API partially, and I know I needed at least one of those methods.